crates/view/src/storage.rs

```diff
@@ -1714,8 +1715,29 @@ impl Storage {
             pool.get()?
                 .prepare_cached(&q)?
                 .query_and_then([], |row| {
-                    let position_id: Vec<u8> = row.get("position_id")?;
-                    Ok(position::Id(position_id.as_slice().try_into()?))
+                    let position_id_bytes: Vec<u8> = row.get("position_id")?;
+                    let id = position::Id(position_id_bytes.as_slice().try_into()?);
+
+                    let position_state_str: String = row.get("position_state")?;
```

Comment:
Getting errors that position_state isn't in the database 🤔 , but it's here: https://github.com/penumbra-zone/penumbra/blob/b85801608811f5b14f1dc399628847cce02d78e8/crates/view/src/storage/schema.sql#L129

Reply:
Weirdly, this seems to only fail in smoke tests.

Reply:
> Weirdly, this seems to only fail in smoke tests.

Not just in smoke tests: I can replicate the smoke test failure locally, yes, but I can also run the same cmd locally against a devnet and observe the same failure:

```
❯ cargo run --release --bin pcli -- --home ~/.local/share/pcli-localhost/ tx position close-all
warning: /home/conor/src/penumbra/crates/bin/pindexer/Cargo.toml: `default-features` is ignored for penumbra-app, since `default-features` was not specified for `workspace.dependencies.penumbra-app`, this could become a hard error in the future
    Finished release [optimized] target(s) in 0.28s
     Running `target/release/pcli --home /home/conor/.local/share/pcli-localhost/ tx position close-all`
Scanning blocks from last sync height 628 to latest height 628
[0s] ██████████████████████████████████████████████████       0/0       0/s ETA: 0s
Error: status: Unavailable, message: "error getting positions: Invalid column name: position_state", details: [], metadata: MetadataMap { headers: {"content-type": "application/grpc"} }
```

Reply:
It's at least the `close-all` subcommand that's affected. Currently there are a lot of calls to both `owned_positions` and `owned_position_ids`, and we'd have to sort out all the different ones to get the tests passing again. The smoke test failures are correct, though, in that they accurately indicate breakage of the tested subcommands.